RecoHI/HiTracking/python/HILowPtConformalPixelTracks_cfi.py

```diff
@@ -2,6 +2,7 @@
 
 from RecoTracker.TkHitPairs.hitPairEDProducer_cfi import hitPairEDProducer as _hitPairEDProducer
 from RecoPixelVertexing.PixelTriplets.pixelTripletHLTEDProducer_cfi import pixelTripletHLTEDProducer as _pixelTripletHLTEDProducer
+from RecoTracker.TkMSParametrization.multipleScatteringParametrisationMakerESProducer_cfi import *
```

Comment:
can this be instead included in `CalibTracker/Configuration/python/Tracker_DependentRecords_forGlobalTag_nofakes_cff.py` ?
@cms-sw/alca-l2 please clarify if there would be concerns about it.
I see that there we already include `RecoLocalTracker.SiStripRecHitConverter.StripCPEfromTrackAngle_cfi`, which has a superset of dependencies compared to MultipleScatteringParametrisationMakerESProducer.cc

Reply:
Fine for me if @cms-sw/alca-l2 agrees.

Reply:
We are a bit reluctant, to me it seems like `MultipleScatteringParametrisationMakerESProducer` is not really what belongs to `Tracker_DependentRecords_forGlobalTag_nofakes_cff`. Of course I understand that `StripCPEfromTrackAngle_cfi ` already includes it, but that's more a side-effect then the original purpose of that file. Can you please elaborate why you think it should be there instead of as it is now?

Reply:
I understand, tho I still don't see how `MultipleScatteringParametrisation` is a `tracker dependent record`. In the sense that the tracker material budget is influencing the multiple scattering?

Reply:
>  I still don't see how `MultipleScatteringParametrisation` is a `tracker dependent record`. In the sense that the tracker material budget is influencing the multiple scattering?

yes, it's derived per `DetLayer` of the tracker; the relationship is built via `MultipleScatteringGeometry(const GeometricSearchTracker &`; the additional dependence on the magnetic field comes in through one of the methods that derives related set of layers (in this limited sense this hints to navigation).

Reply:
Ok Slava, I think this is convincing enough for us, no objections anymore from the AlCa side